### PR TITLE
SQL commands PR

### PR DESCRIPTION
**Pull request by faridmustafayev**

### CODE REVIEW

File: src/main/resources/test.sql [hunk 1]
```diff
@@ -0,0 +1,74 @@
+CREATE TABLE authors
+(
+    id         SERIAL PRIMARY KEY,
```
Comment (turqudquliyev) [inline src/main/resources/test.sql]: use BIG SERIAL instead of SERIAL for id field in all tables

File: src/main/resources/test.sql [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    first_name VARCHAR(128),
+    last_name  VARCHAR(128),
+    email      VARCHAR(256),
+    created_at TIMESTAMP
+);
+
+CREATE TABLE posts
+(
+    id         SERIAL PRIMARY KEY,
+    title      VARCHAR(255),
```
Comment (turqudquliyev) [inline src/main/resources/test.sql]: size must be 2^n (64,128,256 and etc), and apply all tables

File: src/main/resources/test.sql [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    update_at  TIMESTAMP
+);
+
+ALTER TABLE authors
+    ADD COLUMN about TEXT;
+
+ALTER TABLE authors
+    ADD COLUMN nick VARCHAR(128);
+
+CREATE TABLE author_posts
```
Comment (turqudquliyev) [inline src/main/resources/test.sql]: rename to authors_posts

File: src/main/resources/test.sql [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    ADD CONSTRAINT authors_nick_email_key UNIQUE (nick, email);
+
+ALTER TABLE posts
+    ADD COLUMN image_url TEXT;
+
+CREATE TABLE tags
+(
+    id         SERIAL PRIMARY KEY,
+    tag        VARCHAR(255),
+    created_at VARCHAR
```
Comment (turqudquliyev) [inline src/main/resources/test.sql]: created_at must be TIMESTAMP

File: src/main/resources/test.sql [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+);
+
+CREATE INDEX idx_tag ON tags (tag);
+
+CREATE TABLE post_tags
```
Comment (turqudquliyev) [inline src/main/resources/test.sql]: posts_tags

**Comment by turqudquliyev** [comment]
use amend commit

**Comment by turqudquliyev** [comment]
commentleri solve edib merge ede bilersen :)

**Comment by faridmustafayev** [comment]
> use amend commit

**Comment by faridmustafayev** [comment]
> use amend commit

I can't use amend commit. The reason is that my last commit is applicaton.yaml, so I did force push by creating a fake commit

**Comment by faridmustafayev** [comment]
> commentleri solve edib merge ede bilersen :)

I merged. Thanks for your commits

**Pull request merged**